qiskit/tools/wigner_function_visualisation.py

```diff
@@ -0,0 +1,264 @@
+"""
+
+Wigner Function Visualization Module
```

Comment:
These functions should be moved inside the `visualization.py` module rather than be seperate.

I also think it makes more sense to rename the `wigner_function` function to `plot_wigner_function`, and to rename what you've called `plot_wigner_function` to `plot_wigner_data` since it plots the output from the `wigner_data` function in the tomography module.